core/src/main/scala/kanaloa/reactive/dispatcher/queue/QueueProcessor.scala

```diff
@@ -65,7 +65,6 @@ class QueueProcessor(
 
     case Terminated(worker) if workerPool.contains(worker) ⇒
       removeWorker(worker)
-      healthCheck()
```

Comment:
Correct, the system will still try to recreate workers continuously but not constantly any more.